docs/content/development.md

```diff
@@ -0,0 +1,199 @@
+---
+title: Development
+kind: contrib
+weight: 2
+---
+
+OPA is written in the [Go](https://golang.org) programming language.
+
+If you are not familiar with Go we recommend you read through the [How to Write Go
+Code](https://golang.org/doc/code.html) article to familiarize yourself with the standard Go development environment.
+
+Requirements:
+
+- Git
+- GitHub account (if you are contributing)
+- Go (version 1.15+ is supported though older versions are likely to work)
+- GNU Make
+
+## Getting Started
+
+After cloning the repository, just run `make`. This will:
+
+- Build the OPA binary.
+- Run all of the tests.
+- Run all of the static analysis checks.
+
+If the build was successful, a binary will be produced in the top directory (`opa_<OS>_<ARCH>`).
+
+Verify the build was successful with `./opa_<OS>_<ARCH> run`.
+
+You can re-build the project with `make build`, execute all of the tests
+with `make test`, and execute all of the performance benchmarks with `make perf`.
+
+The static analysis checks (e.g., `go fmt`, `golint`, `go vet`) can be run
+with `make check`.
+
+> To correct any imports or style errors run `make fmt`.
+
+## Workflow
+
+### Fork, clone, create a branch
```

Comment:
We had a list here, but that suffered from https://github.com/gohugoio/hugo/issues/2928. So, reformatted this slightly...